src/content/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors.mdx

```diff
@@ -33,29 +33,36 @@ For a look at how to use the Selenium IDE to record your workflows and export th
 
 ## Visit a URL [#visiting-a-url]
 
-All scripts begin by specifying which URL the monitor should navigate to. To specify a URL, call `$browser.get("url")`:
+All scripts begin by specifying which URL the monitor should navigate to. To specify a URL, call `$webDriver.get("url")`:
 
 ```
-$browser.get("https://mywebsite.com");
+await $webDriver.get("https://mywebsite.com");
 ```
 
 ## Sequence actions [#sequence]
 
-Because WebDriverJS is asynchronous, scripting actions can sometimes execute out of order. To force script actions to execute in order, wrap each action in a `then(function(){})` call:
+Because WebDriverJS is asynchronous, scripting actions can sometimes execute out of order. To force script actions to execute in order, use the `await` keyword:
 
 ```
-$browser.get("https://my-website.com").then(function(){
-    return $browser.findElement($driver.By.linkText("Configuration Panel"));
+await $webDriver.get("https://my-website.com");
+await $webDriver.findElement($selenium.By.linkText("Configuration Panel"));
+```
+
+You can also wrap each action in a `then(function(){})` call, but in that case the wrapped function must `return` each asynchronous function to ensure they complete before the script moves on: 
+
+```
+$webDriver.get("https://my-website.com").then(function(){
+    return $webDriver.findElement($selenium.By.linkText("Configuration Panel"));
```

Comment:
Very not sure if this should be. left in or not. Technically still a valid approach as long as the Promises are returned?